runtimes/python/python_apps.md

```diff
@@ -53,23 +53,19 @@ Remove the environment variable or set it to `false` to disable this feature.
 If you have any question, feel free to [contact our support](https://www.clever-cloud.com/doc/get-help/support/).
 
 
-## Configuration file
+## Configuring the deployment
```

Comment:
I'd like to use the term `deployment` less (as long as we've not properly defined it)